docs/getting_started.md

```diff
@@ -251,19 +235,20 @@ option = st.sidebar.selectbox(
 ```
 
 Most of the elements you can put into your app can also be put into a sidebar using this syntax:
-`st.sidebar.[element_name]()`. Here are a few examples that show how it's used: `st.sidebar.markdown()`, `st.sidebar.slider()`, `st.sidebar.line_chart()`.
+`st.sidebar.[element_name]()`, including:
 
-The only exceptions right now are `st.write` (you
-should use `st.sidebar.markdown()` instead), `st.echo`, and `st.spinner`. Rest
-assured, though, we're currently working on adding support for those too!
+- `st.sidebar.markdown()`
+- `st.sidebar.slider()`
+- `st.sidebar.line_chart()`
 
-## Show progress
+Currently, the only exceptions to using widgets in the sidebar right now are `st.write` (you
```

Comment:
`Currently` and `right now` are saying the same thing. Suggest removing one. 

- Currently, the only exceptions to using widgets in the sidebar are `st.write`
- The only exceptions to using widgets in the sidebar right now are `st.write`

Reply:
:+1: